applicatie/static/js/custom/bulk.js

```diff
@@ -31,13 +31,108 @@ function initialiseer_form_controls_bulkcorrecties() {
 
 function bulkcorrecties_verwerken() {
 
-    let tekst = "Weet u zeker dat u de ingevoerde correcties wilt verwerken?\r\n"
-    tekst = tekst + "Zo ja,  klik op OK\r\n"
-    tekst = tekst + "Zo nee, klik op Annuleren (Cancel)"
+    let form_bulk = $("#" + FORMULIER_BULK);
+
+    let bulkcorrecties = form_bulk.find("div[id^='bulkcorr_']");
+
+    let formulier_correct_ingevuld = true;
+
+    let aantal_correcties = 0;
+
+    const _q = "\"";
+    const _a = "\{";
+    const _z = "\}";
+
+    // JSON string openen met {
+    let _bulk = _a;
```

Comment:
Is er een reden voor underscore aan het begin van een variabele of een speciale betekenis in JavaScript?

Reply:
nou ja, javascript variabelen **mogen** in ieder geval wel met een underscore beginnen, maar dat heeft geen speciale betekenis. Soms wordt een underscore voor een variabele gebruikt om aan te geven dat de betreffende variabele "private" is (bijvoorbeeld in een class), maar dat wordt niet afgedwongen volgens mij. In dit geval zit er geen diepe gedachte achter behalve dat ik het zelf wel leesbaar vond.

Reply:
Dit is nu weg, ik heb de functie gerefactored.

Reply:
ok! vroeg het me gewoon af, voor mij impliceert een underscore een soort speciale betekenis.. als we het gebruiken zou ik fan zijn van de underscore slechts te gebruiken voor private variabelen van classes inderdaad en hier dus niet.